include/zephyr/usb/class/usb_audio.h

```diff
@@ -100,39 +100,81 @@ enum usb_audio_fucs {
  * Refer to Table 2-1 - Table 2-4 from termt10.pdf
  */
 enum usb_audio_terminal_types {
-	/* USB Terminal Types */
+	/**
+	 * @name USB Terminal Types
```

Comment:
is name needed here? this is already within an enum

Reply:
I mimicked the existing comments "USB Terminal Types", "Input Terminal Types", "Output Terminal Types", ... to keep (what I think was) the original intent of grouping those three sub-types of terminal types. But now that you're mentioning it, it doesn't look like `@name`s within an enum actually do anything. Will update as part of addressing https://github.com/zephyrproject-rtos/zephyr/pull/60533#pullrequestreview-1543842539 which I had missed